README.md

```
@@ -1 +1,282 @@
+
+1. **Clone the repository**
+```bash
+git clone <repository-url>
+cd apatye_backend
+```
+
+2. **Create environment file**
+```bash
+cp .env.example .env
+# Edit .env with your configuration
+```
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)